docs/src/configuration/characters.md

```diff
@@ -9,7 +9,7 @@ Configurable fields:
 - `base_size`: Size of the character for collision math
 - `base_health`: Base health of the character
 - `skills`: This is a map of integer (as string) to skills, the integers represent the id and ordering of skills for calling them
-- `max_inventory_size`: Maximum amount of loots that the character can hold in their inventory
+- `burst_loads`: Number of times the character's basic skill can be used in a row before going on cooldown
```

Comment:
We're still keeping the `:max_inventory_size` field in the config. Why are we deleting it from the docs?

Reply:
It was a mistake, now it should be restored!